modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -73,15 +73,23 @@ For more information about filesystem and partitioning details, see xref:install
 Add 100 MB per {suse} product, 1 GB per {redhat} or other product.
 Consider to double the space if the server is used for Inter-Server Synchronization (ISS)
 
+|
+| [path]``/var/lib/containers/storage/volumes/srv-www``
+| Minimum 100 GB
+
+* Storage requirments should be calculated for the number of ISO distribution images, containers, and bootstrap repositories you will use.
+
```

Comment:
This is getting confusing and then we have bug reports like this https://bugzilla.suse.com/show_bug.cgi?id=1230502

I think what we should do is to scrap all these subvolumes and focus just on container volume dir requirements:

[path]``/var/lib/containers/storage/volumes`` should be 150GB+ depending on number of products.

And optionally have [path]``/var/lib/containers/storage/volumes/var-pgsql`` with that 50GB.

Reply:
ok, change accordingly.